### ISSUE TITLE
Integrate docker-compose within docker cli

### ISSUE THREAD

**Issue by fdellabetta** [docker/compose#4969]
Feature request:
Is it possible to embed docker-compose tool into the docker CLI ?

E.g. 
Could it be
```
$ docker compose [-f <arg>...] [options] [COMMAND] [ARGS...]
```
instead of 
```
$ docker-compose  [-f <arg>...] [options] [COMMAND] [ARGS...]
```

**Comment by shin-** [comment]
That will never happen in the form you describe - they're entirely different codebases written in 2 different languages.

But `docker stack` is a thing: https://docs.docker.com/engine/reference/commandline/stack/

**Comment by fdellabetta** [comment]
I see, but let me rephrase the feature request like this:

Is it possible to user docker stack without swarm (and maybe call it docker compose) ?

**Comment by shin-** [comment]
The first part of my answer covers that aspect.

**Comment by fdellabetta** [comment]
Thank you. I will close the issue, then

**Comment by Delapouite** [comment]
> That will never happen in the form you describe 

To help people stumbling upon this thread coming from a search engine.
It did finally happen, many years later : https://docs.docker.com/compose/migrate/
